.circleci/config.yml

```diff
@@ -11,6 +11,32 @@ parameters:
     description: Go Version to use
 
 jobs:
+
+  build-docker-image:
+    docker:
+      - image: cimg/go:<< pipeline.parameters.gover >>
+
+    environment:
+      DOCKER_IMAGE_NAME: jcid/stackexchange-dnscontrol
+      DOCKER_IMAGE_VERSION: 3.18.1
```

Comment:
Is there a Circle CI variable available with the DNSControl version number?

Reply:
Not that I know of.   @cdhunt do?

You could extract it from main.go:

    grep -E "Version *= \"" main.go  | awk '{ print $3 }' | tr -d \"

Reply:
Yes! Discussed that this can't be tested now because Circle CI is blocked.

<img width="1280" alt="circle-ci-blocked" src="https://user-images.githubusercontent.com/1150425/184139032-35d6ce31-3575-4e9a-894c-0644a5fdd635.png">